mycroft/skills/intent_service_interface.py

```diff
@@ -216,7 +216,7 @@ def __init__(self, bus=None, timeout=5):
         self.bus = bus
         self.timeout = timeout
 
-    def get_adapt_intent(self, utterance, lang="en-us"):
+    def get_adapt_intent(self, utterance, lang=None):
```

Comment:
should we try to read language from messages using `dig_for_message` here? Since we're generating new `Message` objects in these methods, this is the last place we have a chance of resolving the originating message

We could make a decorator for this that could be applicable to every other method that looks up user language